modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java

```diff
@@ -187,6 +191,57 @@ public void testPostExportTaskWithFilter() throws Exception {
 			JSONCompareMode.LENIENT);
 	}
 
+	@Test
+	public void testPostExportTaskWithNestedFieldNames() throws Exception {
+		ObjectEntry objectEntry = ObjectEntryTestUtil.addObjectEntry(
+			objectDefinition, OBJECT_FIELD_NAME_TEXT, "TestObject");
+
+		ObjectDefinition objectDefinition2 =
+			ObjectDefinitionTestUtil.publishObjectDefinition(
```

Comment:
I think you don't need a second object definition. You can test this with other existing fields, such as creator

Reply:
(for example, by including `creator.id`)